modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -61,18 +65,17 @@ def submit_power_of_attorney(poa_code)
 
           ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id)
 
-          ClaimsApi::VBMSUpdater.perform_async(power_of_attorney.id) if enable_vbms_access?
+          ClaimsApi::PoaVBMSUpdater.perform_async(power_of_attorney.id) if enable_vbms_access?
 
           # This builds the POA form *AND* uploads it to VBMS
           ClaimsApi::PoaFormBuilderJob.perform_async(power_of_attorney.id)
```

Comment:
this job calls `PoaUpdater` and ultimately that calls `PoaVBMSUpdater` dependent on a value. we're duplicating calls here. 👀 Since we'll build and upload the form for our consumers, I think we're safe to remove the above two jobs and only use the `PoaFormBuilderJob` - be worth taking a second look at though.

Reply:
interesting as well. We call MPI to get Veteran data, and we can use that data to find the File Number for uploading to VBMS. 

Our goal was to initially submit the document through Benefits Documents, but now it looks like we may have a limiting factor for a successful submission, so let's keep our flow built to upload to VBMS. 

Ultimately we should probably look at reworking the logic internal `PoAFormBuilderJob` to use the birls file number pulled from MPI so we don't have the make the additional call to BGS in `retrieve_veteran_file_number`. This is more "future thought" than work needed in this PR. That will impact v1 and v2 though. 🤔

Reply:
I'm starting to wonder if we want a new upload job for v2 entirely. we could pull `power_of_attorney.auth_headers['va_eauth_birlsfilenumber']` as the file number and remove the call we have from bgs-ext - would also prevent the need for that PR... this would allow us to safely implement and test it before we just switch v1's job..... plus we need to find a way to bypass the signatures being blank for only v2 as mentioned above. 

creating a new service to inherit the pdf build job and just overwriting the `page2_signatures` method would be easy, but creating a new sidekiq job and ultimately a new `upload_to_vbms` method from within the external service (`PoaVbmsSidekiq`) to simply replace the bgs-ext call for a file number gets a bit... convoluted.  actually it may not 🤔 ... we just inherit from a new job and replace which service it's calling for the upload.. but then another service to override that method.. ok, so it is getting a little convoluted. 

thoughts? always open to other ideas and just trying to make sure we implement meaningful changes.

Reply:
I was actually thinking we could base this off of CCG / ACG pretty safely. V1 "supports" CCG but we have no consumers using it that way. Technically we could look for `@is_valid_ccg_flow` and know it's from v2? with that in mind we could bypass it for now with that????